tests/network/connectivity/test_pod_network.py

```diff
@@ -75,10 +76,8 @@ def cloud_init_ipv6_network_data(dual_stack_network_data):
 
 @pytest.mark.gating
 @pytest.mark.polarion("CNV-2332")
-def test_connectivity_over_pod_network(
+def test_connectivity_over_pod_ipv4_network(
```

Comment:
nit
```suggestion
def test_connectivity_over_pod_network_ipv4(
```
The change improves readability by keeping "pod network" together as a common term and making "ipv4" a clear qualifier.